benchmarking/run_all_benchmarks_on_slurm.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+sbatch --job-name=bc_seals_ant run_benchmark_on_slurm.sh train_imitation bc seals_ant
```

Comment:
Can replace the whole file here also in a similar way as above:

```bash
#!/bin/bash

envs=(
    "seals_ant"
    "seals_half_cheetah"
    "seals_hopper"
    "seals_swimmer"
    "seals_walker"
)

script_algos=(
    "imitation" "bc"
    "imitation" "dagger"
    "adversarial" "airl"
    "adversarial" "gail"
)

for env in "${envs[@]}"; do
    for ((i=0; i<${#script_algos[@]}; i+=2)); do
        script=${script_algos[$i]}
        algo=${script_algos[$((i+1))]}
        job_name="${algo}_${env}"
        echo sbatch --job-name="${job_name}" run_benchmark_on_slurm.sh "train_${script}" "${algo}" "${env}"
    done
done

Reply:
See above comment.